### DISCUSSION TITLE
Outside discord

### DISCUSSION THREAD

**Discussion by GuPoroca** [lavalink-devs/Lavalink#969] [Q&A] [answered]
Can I use Lavalink to play songs in a webpage?, that is, outside of discord?

**Comment by duncte123** [comment, marked as answer]
Unfortunately not. Lavalink is built to send audio to discord and take the load away from bots.

**Reply by topi314** [reply]
I want to add that [lavaplayer](https://github.com/lavalink-devs/lavaplayer) is totally capable of doing this
